src/pages/regulatory/index.js

```diff
@@ -381,44 +357,79 @@ const Regulatory = () => {
                                     components={[
                                         <LinkText
                                             key={0}
                                             weight="bold"
                                             target="_blank"
-                                            href="/regulatory/Deriv_(FX)_Ltd.pdf"
+                                            href="/regulatory/Deriv_(V)_Ltd.pdf"
+                                            rel="noopener noreferrer"
+                                        />,
+                                        <LinkText
+                                            key={1}
+                                            weight="bold"
+                                            target="_blank"
+                                            href="/regulatory/Champion_Group_Ltd.pdf"
                                             rel="noopener noreferrer"
                                         />,
                                     ]}
                                 />
                             </Text>
-                            <Text mt="2rem" max_width="58.8rem">
+                            <Text max_width="58.8rem">
+                                {localize(
+                                    'Clients in the rest of the world (except for restricted countries such as the USA, Canada, and Hong Kong) who wish to trade CFDs on financial instruments can have MetaTrader 5 accounts under Deriv (V) Ltd and Champion Group Ltd.',
+                                )}
+                            </Text>
+                        </CssGridColumn>
+
+                        <ResponsiveGrid align="flex-start">
+                            <Mobile>
+                                <StyledHeader lh="4rem" as="h4" mb="1.6rem">
+                                    {localize('Deriv (SVG) LLC')}
+                                </StyledHeader>
+                            </Mobile>
+                            <SVG />
+                        </ResponsiveGrid>
+                        <CssGridColumn>
+                            <Desktop>
+                                <StyledHeader lh="4rem" as="h4">
+                                    {localize('Deriv (SVG) LLC')}
+                                </StyledHeader>
+                            </Desktop>
+                            <Text mt="0.8rem" mb="1.6rem" max_width="58.8rem">
                                 {localize(
-                                    'Clients from the rest of the world (except for restricted countries such as the USA, Canada, and territories such as the European Union) who wish to trade CFDs on forex and cryptocurrencies will have their MetaTrader 5 accounts opened under Deriv (FX) Ltd.',
+                                    'Deriv (SVG) LLC, Hinds Buildings, Kingstown, St. Vincent and the Grenadines (company number 273 LLC 2020).',
```

Comment:
I think the  sentence  
`company number 273 LLC 2020`
should be in bold and use Localize

Reply:
I gotta check to see if there is a pdf file for this link..thanks for pointing it out 🙏

Reply:
the ` company number 273 LLC 2020` 
still missing the bold styling